src/contracts/interfaces/IPaymentCoordinator.sol

```diff
@@ -0,0 +1,162 @@
+
+    /// EXTERNAL FUNCTIONS ///
+
+    /**
+     * @notice Initializes the contract
+     * @param initialOwner The address of the initial owner of the contract
+     * @param _activationDelay Delay in timestamp before a posted root can be claimed against
+     * @param _globalCommissionBips The commission for all operators across all avss
+     * @dev Only callable once
+     */
+    function initialize(address initialOwner, uint32 _activationDelay, uint16 _globalCommissionBips) external;
+    
+    /**
+     * @notice Creates a new range payment on behalf of an AVS, to be split amongst the
+     * set of stakers delegated to operators who are registered to the `avs`
+     * @param rangePayment The range payment being created
+     * @dev Expected to be called by the ServiceManager of the AVS on behalf of which the payment is being made
+     * @dev The duration of the `rangePayment` cannot exceed `MAX_PAYMENT_DURATION`
+     * @dev The tokens are sent to the `claimingManager` contract
+     * @dev This function will revert if the `rangePayment` is malformed,
+     * e.g. if the `strategies` and `weights` arrays are of non-equal lengths
+     */
+	function payForRange(RangePayment calldata rangePayment) external;
+
+    /**
+     * @notice similar to `payForRange` except the payment is split amongst *all* stakers
+     * rather than just those delegated to operators who are registered to a single avs
+     */
+    function payAllForRange(RangePayment calldata rangePayment) external;
+
+
+    /**
+     * @notice Sets the delay in timestamp before a posted root can be claimed against
+     * @param _activationDelay Delay in timestamp before a posted root can be claimed against
+     * @dev Only callable by the contract owner
+     */
+    function setActivationDelay(uint32 _activationDelay) external;
+
+    /**
+     * @notice Sets the global commission for all operators across all avss
+     * @param _globalCommissionBips The commission for all operators across all avss
+     * @dev Only callable by the contract owner
+     */
+    function setGlobalOperatorCommission(uint16 _globalCommissionBips) external;
+
+    /**
+     * @notice Sets the address of the entity that can claim payments on behalf of the account
+     * @param account The account whose recipient is being set
+     * @param recipient The address of the entity that can claim payments on behalf of the account
+     * @dev Only callable by the account or their paymentRecipient
+     */
+    function setRecipient(address account, address recipient) external;
+
+    /**
+     * @notice Creates a new distribution root
+     * @param root The merkle root of the distribution
+     * @param paymentsCalculatedUntilTimestamp The timestamp until which payments have been calculated
+     * @dev Only callable by the paymentUpdater
+     */
+    function submitRoot(bytes32 root, uint32 paymentsCalculatedUntilTimestamp) external;
```

Comment:
might.... get rid of the `paymentsCalculatedUntilTimestamp`. Also my spec includes a check against beacon state roots. are we going to do that?

Reply:
I'm fine with this for now
will talk to you about the beacon state root part -- I definitely didn't see that, myself